docs/deciding_what_data_to_stream_with_dfe_analytics.md

```diff
@@ -0,0 +1,70 @@
+
+## How do we exclude data from streaming?
+Data for which DfE has no basis to store or process **must not** be streamed into BigQuery by ```dfe-analytics```.
+
+This data may take the form of individual fields within a table in an application's database, or entire tables.
+
+To exclude these fields, remove them from ```config/analytics.yml``` and add them to ```config/analytics_blocklist.yml```.
+
+For more information see [here](https://github.com/DFE-Digital/dfe-analytics/tree/SL-dev?tab=readme-ov-file#5-send-database-events).
+
+## How long should we keep data for?
+Data processed under the 'public task' basis for processing described in the previous section may continue to be stored and processed for as long as it is needed for any of the public task(s) that is stored for.
+
+This means that once it is no longer useful for any such task it **must** be deleted. For example:
+- Data to support analysis of teachers' progression throughout their careers so that the effectiveness of policy measures to retain teachers can be evaluated might need to be retained for 50 years
+- Data that is only stored to support evaluation of the impact of policy changes on ITT recruitment might need to be stored for a number of years (say, 7) but not longer
+- Data that is only stored to support evaluation of the immediate effects of a design or content change to a digital service might only need to be stored for around a year
+
+IAOs and their teams should also consider public tasks which may need to be performed by other teams within DfE, both now and in the future, when evaluating how long data should be retained for.
+
+Teams **should** consider [implementing a data retention schedule](https://github.com/DFE-Digital/dfe-analytics-dataform/?tab=readme-ov-file#data-retention-schedules) in ```dfe-analytics-dataform``` to ensure that data streamed by ```dfe-analytics``` is deleted automatically once it is no longer required.
+
+## How do we hide more sensitive data when it is streamed?
+Data that is more sensitive than most personal data **should** be marked as such when it is streamed into BigQuery by ```dfe-analytics``` - if it is streamed at all. This includes:
+- Human-readable PII, such as names, email addresses, home addresses and TRNs
```

Comment:
This is a really important point.
Can we include a definition of what is Human-readable PII, as well as a complete list of theses types of data? I'm assuming it not in the 100's or 1000's of types of PII....

Reply:
It's not possible to be exhaustive because any ID a service decides to give out to its users (i.e. not just an internal ID) is human readable PII. But I've expanded the list as much as I can!